apps/civil/civil-service/aat-image-policy.yaml

```diff
@@ -2,14 +2,6 @@ apiVersion: image.toolkit.fluxcd.io/v1beta2
 kind: ImagePolicy
 metadata:
   name: aat-civil-service
-  annotations:
-    hmcts.github.com/prod-automated: disabled
 spec:
-  filterTags:
-    pattern: '^pr-6489-[a-f0-9]+-(?P<ts>[0-9]+)'
-    extract: '$ts'
```

Comment:
you can keep this for now, just change when you need a new image at later stage

Reply:
updated